### DISCUSSION TITLE
How to import the matrix suction distribution results from the FEM into MPM modeling

### DISCUSSION THREAD

**Discussion by LongTIme1111** [Anura3D/Anura3D_OpenSource#213] [Q&A]
Hi all
  I am simulating a landslide caused by rainfall, which lasts for several days, and the calculation may be caused by too long a time due to the use of MPM, so I want to use finite element software (similar to Geostudio) to simulate the critical state of instability, and import the critical state into MPM to observe the sliding state, how can this method be implemented?

**Comment by FrancescaCeccato** [comment]
Yes, this option will be available is version 2024 that is going to be released on June 28! Stay tuned!
